test/boostTest.cpp

```diff
@@ -21,4 +21,8 @@
  */
 
 #define BOOST_TEST_MODULE EthereumTests
+#pragma GCC diagnostic push
+#pragma GCC diagnostic ignored "-Wunused-parameter"
```

Comment:
Oh so this is for boost testing? I thought it was for some parameter inside your tests. My bad then. 
But out of curiosity then, with other compilers, say MSVC you do still get the warnings, yes?